dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -115,11 +115,22 @@ def main():
     collector_cls = config.Config.get_collector(collector['name'])
     collector_objects.extend(collector_cls.launch_collector(**new_args))
 
+  # Set global errors for final reporting
+  global_errors = []
+
   # Wait for collectors to finish and collect output
   collector_output = []
+  errors = []
   for collector_obj in collector_objects:
     collector_obj.join()
     collector_output.extend(collector_obj.results)
+    if collector_obj.errors:
+      err = (collector_obj.__class__.__name__, ", ".join(collector_obj.errors))
```

Comment:
Lets add proper name attributes for all these modules. This ended up being a pain in Plaso, so lets try and avoid making the same mistakes if we can. Fine to do in a followup CL though.